src/client.rs

```diff
@@ -114,40 +108,55 @@ macro_rules! define_rpc_args {
         $(
             paste! {
                 #[derive(Parser, Debug, Clone, PartialEq, Eq)]
+                #[clap(group(ArgGroup::new(concat!(stringify!($prefix), "_jwt")).required(true).multiple(false).args(&[
+                    concat!(stringify!($prefix), "_auth_jwtsecret_path"),
+                    concat!(stringify!($prefix), "_auth_jwtsecret")])))
```

Comment:
Can we add another group here for rpc jwt that is not required but enforces either the secret or the path?